src/nanoemoji/write_font.py

```diff
@@ -573,45 +573,45 @@ def _colr_ufo(
     ufo: ufoLib2.Font,
     color_glyphs: Tuple[ColorGlyph, ...],
 ):
+    black = Color(0, 0, 0, 1.0)
+
     # We want to mutate our view of color glyphs
     color_glyphs = list(color_glyphs)
 
     # We only store opaque colors in CPAL for COLRv1, as 'alpha' is
     # encoded separately.
-    colors = set(
+    cpal_colors = set(
         c if colr_version == 0 else c.opaque()
         for c in chain.from_iterable(g.colors() for g in color_glyphs)
         if not c.is_current_color()
     )
-    # Sort colors so the index into colors == index into CPAL palette.
-    # Try to keep the original palette entry index if present.
-    iter_colors_without_index = iter(sorted(c for c in colors if c.index is None))
-    colors_by_index = {}
-    for c in colors:
+
+    # Chrome 98 doesn't like when COLRv1 font has empty CPAL palette so make sure we have at least one
+    # TODO(anthrotype): File a bug and remove hack once the bug is fixed upstream
+    if not cpal_colors:
+        cpal_colors = {black}
+
+    # We need enough slots for all the colors, or to reach the max index, whichever is greater
+    cpal_slots = max(len(cpal_colors), max(c.index or -1 for c in cpal_colors))
```

Comment:
indices start counting at 0, so to compute the length of the final list you need to add +1 to the maximum index

```suggestion
    cpal_slots = max(len(cpal_colors), max(c.index or -1 for c in cpal_colors) + 1)
```